tests/testCoadds.py

```diff
@@ -65,15 +66,21 @@
 from lsst.pipe.tasks.multiBand import (DetectCoaddSourcesTask, MergeDetectionsTask,
                                        MeasureMergedCoaddSourcesTask, MergeMeasurementsTask)
 
-try:
-    REUSE_DATAREPO
-except NameError:              # Only set variables on initial import
-    REUSE_DATAREPO = True      # If mocks are found (for each test), they will be used instead of regenerated
-    CLEANUP_DATAREPO = True    # Delete mocks after all tests (if REUSE_DATAREPO) or after each one (else).
-    DATAREPO_ROOT = "testCoadds-data"
+REUSE_DATAREPO = True  # retain the data repo for each test (which greatly speeds up the test)?
+SAVE_DATAREPO = True  # retain data repo after the test finishes? Only set true for debugging
```

Comment:
should this be submitted to master set to False then (from looking at the comment)